mkroosevelt.js

```diff
@@ -94,8 +94,12 @@ function createSampleApp() {
     console.error('There was an error in copying the sample app: ' + err.message);
   }
 
-  if (fs.existsSync(path.normalize(cmd + '/.npmignore'))) {
-    fs.renameSync(path.normalize(cmd + '/.npmignore'), path.normalize(cmd + '/.gitignore')); // fix to compensate for this 'feature' https://github.com/npm/npm/issues/1862
+  try {
+    fs.accessSync(path.normalize(cmd + '/.npmignore'));
+    fs.renameSync(path.normalize(cmd + '/.npmignore'), path.normalize(cmd + '/.gitignore')); // fix to compensate for this "feature" https://github.com/npm/npm/issues/1862
+  }
+  catch (err) {
+    // If we do error in the try is doesn't matter because we are only correcting for that "feature". The sample app will still copy correctly, if this errors.
```

Comment:
there really is no point to even bothering with a try/catch block if you're going to do nothing with the error